src/graphics.rs

```diff
@@ -839,7 +840,7 @@ impl Graphics {
         let quad_bindings = mq::Bindings {
             vertex_buffers: vec![quad_vertices, instances],
             index_buffer: quad_indices,
-            images: vec![*null_texture],
+            images: vec![(*null_texture).texture],
```

Comment:
should not be necessary to deref `null_texture` here, derefs are implicit in field accesses